### PR TITLE
Adds isBuild check in html for dev js files

### LINKED ISSUE #2: index.html resulting from npm run build contains both file references with hashs and without
I would expect the regular references to be replaced by the ones with hashes.

* Running on windows
* No warnings outputted when running the build command

### PR DESCRIPTION
**Pull request by lukas-reineke**
fixes #2

### CODE REVIEW

**Comment by piotrwitek** [comment]
Hello @lukas-reineke, I'll test it later today.
Thanks!

**Comment by lukas-reineke** [comment]
I could also change the filetype from html to ejs. Wouldn't make a difference in how it works, but might be nicer for highlighting / code-completion in IDE's / text-editors.

**Comment by piotrwitek** [comment]
Thanks but HTML is fine, I believe in standards :)

File: configs/webpack.config.js [hunk 1]
```diff
@@ -156,6 +156,10 @@ module.exports = (env = {}) => {
       new webpack.optimize.CommonsChunkPlugin({
         name: 'manifest',
       }),
+      new HtmlWebpackPlugin({
+        template: './index.html',
+        isBuild: isBuild
```
Comment (piotrwitek) [inline configs/webpack.config.js]: isBuild can be removed

File: index.html [hunk 2]
```diff
@@ -7,9 +7,11 @@
     </head>
     <body>
         <div id="root"></div>
-        <!-- webpack bundles -->
-        <script src="/manifest.js"></script>
-        <script src="/vendor.js"></script>
-        <script src="/app.js"></script>
+        <% if (!htmlWebpackPlugin.options.isBuild) { %>
```
Comment (piotrwitek) [inline index.html]: this entire block can be safely removed

**Review by piotrwitek — CHANGES_REQUESTED** [review]
I tested it and it turns out that only the single change to move HtmlWebpackPlugin to common plugins in config is sufficient, the isBuild and template block is not needed, plugin will do the same automatically. Would you like to apply the requested changes to finish the PR?

**Comment by lukas-reineke** [comment]
True, didn't even noticed that.
Should be fine now.

**Review by piotrwitek — APPROVED** [review]

**Comment by piotrwitek** [comment]
Great, thanks for helping to figure this one out! :)

**Pull request merged**